src/bindgen/parser.rs

```diff
@@ -265,16 +269,25 @@ impl<'a> Parser<'a> {
             self.cache_src.get(&owned_mod_path).unwrap().clone()
         };
 
-        let mod_dir = mod_path.parent().unwrap();
+        // Compute module directory according to Rust 2018 rules
+        let mod_dir_2018;
+
+        let mod_dir = if depth == 0 {
+            mod_path.parent().unwrap()
+        } else {
+            mod_dir_2018 = mod_path.parent().unwrap().join(mod_path.file_stem().unwrap());
```

Comment:
Does this break non-2018 users?

Reply:
It seems to break non-2018 unfortunately, I added a test for that in 0d89c9e and I'll investigate this in more detail.

Reply:
Should be fixed by 835229b